meeshkan/serve/mock/specs.py

```diff
@@ -0,0 +1,43 @@
+import os
+import logging
+from openapi_typed_2 import (
+    convert_to_openapi,
+    OpenAPIObject,
+)
+from typing import List
+import json
+import yaml
+from dataclasses import dataclass
+
+
+@dataclass
+class OpenAPISpecification:
+    """An OpenAPI object with information about the source it was created from."""
+
+    api: OpenAPIObject
+    source: str
```

Comment:
What's an example source?

Reply:
I've added `(such as the file name or URL)` - do you think that is enough?

Reply:
👍